scripts/scripts.js

```diff
@@ -200,6 +200,37 @@ export function formatDate(date) {
   });
 }
 
+function addBlogTopics(main) {
+  const sidebarBolgTopic = main.querySelector('.blog-topics');
```

Comment:
This should be in a block code
* `/blocks/blog-topics/blog-topic.js`
* `/blocks/blog-topics/blog-topic.css` // probably empty

This way you'll receive the reference to the div where to inject the data directly and it will also be done in order, because here you are doing it very early, while the sidebar on mobile, is not even visible to the user.